### ISSUE TITLE
1.6.1version

### ISSUE THREAD

**Issue by yuu2lee4** [andyhu/transliteration#41]
````javascript
/Users/yuri/work/vue-beauty/node_modules/transliteration/data/index.js:1
(function (exports, require, module, __filename, __dirname) { import charmap from './charmap.json';
                                                        
SyntaxError: Unexpected token import
   at createScript (vm.js:53:10)
    at Object.runInThisContext (vm.js:95:10)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/yuri/work/vue-
    beauty/node_modules/transliteration/lib/node/transliterate.js:10:13)
````
version 1.5.4 is ok

**Comment by andyhu** [comment]
Thanks for the spot, I'll inspect it and hopefully fix it soon

**Comment by andyhu** [comment]
@yuu2lee4 can you please let me know your node version?

**Comment by andyhu** [comment]
@yuu2lee4 fixed in version 1.6.2
